### PR TITLE
Release 2 v2 apps (30)

### PR DESCRIPTION
**Pull request by JV0812**
## PLEASE READ

The PR https://github.com/SumoLogic/sumologic-documentation/pull/3376, https://github.com/SumoLogic/sumologic-documentation/pull/3372, and https://github.com/SumoLogic/sumologic-documentation/pull/3426 are merged and created this new single PR.

Following a recent back-end update, all contributors with a local clone or fork of our repository are required to run `yarn install`. This does not apply to [direct page edits](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits).

- [x] Yes, I've run `yarn install`
- [ ] No, does not apply to me

## Purpose of this pull request

This pull request...

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions, .clabot
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

**Comment by jpipkin1** [comment]
@JV0812, I don't recommend using parentheses in your branch names because it can cause problems. For example, when I attempted to check out the branch for this PR, I got an error:
>git checkout Release-2-v2-apps-(30)
>zsh: unknown file attribute: 3

Checkout worked only after I enclosed the branch name in quotation marks:
>git checkout "Release-2-v2-apps-(30)"
>branch 'Release-2-v2-apps-(30)' set up to track 'origin/Release-2-v2-apps-(30)'.
>Switched to a new branch 'Release-2-v2-apps-(30)'

File: docs/integrations/hosts-operating-systems/linux.md [hunk 1]
```diff
@@ -60,64 +56,43 @@ Dec 16 20:26:23 ubuntu sshd[15533]: pam_unix(sshd:auth): authentication failure;
 2016-12-16 19:23:13 remove tomcat7:all 7.0.68-1ubuntu0.1 <none>
 ```
 
-### Sample Queries
+### Sample queries
 
 See [Suggested Searches for Linux OS](#Suggested_Searches_for_Linux_OS).
```
Comment (jpipkin1) [inline docs/integrations/hosts-operating-systems/linux.md:61]:
```suggestion
See [Suggested Searches for Linux OS](#suggested-searches-for-linux-os).
```

File: docs/integrations/microsoft-azure/windows-json.md [hunk 2]
```diff
@@ -15,8 +14,8 @@ The Windows JSON App provides insight into your Windows system's operation and e
 You may also be interested in the [Sumo Logic App for Windows Cloud Security Monitoring and Analytics](docs/integrations/cloud-security-monitoring-analytics/windows.md).
 :::
 
-
-## Log Types
+## Log types
```
Comment (jpipkin1) [inline docs/integrations/microsoft-azure/windows-json.md:17]:
```suggestion
```

Comment (jpipkin1) [reply docs/integrations/microsoft-azure/windows-json.md:17]: The "Log types" heading appeared twice.

File: docs/integrations/microsoft-azure/windows-json.md [hunk 3]
```diff
@@ -79,8 +77,8 @@ _sourceCategory=Labs/windows-jsonformat ( "Audit Policy Change" or "System audit
 | count by msg_summary | sort by _count, msg_summary asc
 ```
 
-
-## Collecting Logs for the Windows JSON App
+## Collecting Logs for the Windows JSON app
```
Comment (jpipkin1) [inline docs/integrations/microsoft-azure/windows-json.md:80]:
```suggestion
```

Comment (jpipkin1) [reply docs/integrations/microsoft-azure/windows-json.md:80]: The "Collecting Logs for the Windows JSON app" heading appeared twice.

File: docs/integrations/saas-cloud/symantec-web-security-service.md [hunk 4]
```diff
@@ -15,11 +15,13 @@ With the Sumo Logic app for Symantec Web Security, you can:
 - **Identify and block suspicious web activity**. You can use the app to monitor suspicious web activity, such as requests to known malicious sites.
 - **Optimize web security policies**. This app provides insights into web traffic that can help you to optimize web security policies. You can also use the app to identify areas where policies may be too restrictive or too permissive and make adjustments accordingly.
 
-## Log Types
+## Log types
```
Comment (jpipkin1) [inline docs/integrations/saas-cloud/symantec-web-security-service.md:18]:
```suggestion
```

Comment (jpipkin1) [reply docs/integrations/saas-cloud/symantec-web-security-service.md:18]: The "Log Types" heading appears twice.

File: docs/integrations/saas-cloud/symantec-web-security-service.md [hunk 5]
```diff
@@ -15,11 +15,13 @@ With the Sumo Logic app for Symantec Web Security, you can:
...
+## Log types
 
 This App uses [Symantec Web Security Service](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/symantec-web-security-service-source) to collect [Access Logs](https://techdocs.broadcom.com/us/en/symantec-security-software/web-and-network-security/cloud-swg/help/wss-reference/accesslogformats-ref.html) from Symantec Web Security Service.
 
-## Sample Log Message
+## Sample log message
```
Comment (jpipkin1) [inline docs/integrations/saas-cloud/symantec-web-security-service.md:23]:
```suggestion
```

Comment (jpipkin1) [reply docs/integrations/saas-cloud/symantec-web-security-service.md:23]: The "Sample log message" heading appears twice.

File: docs/integrations/saas-cloud/symantec-web-security-service.md [hunk 6]
```diff
@@ -64,18 +66,18 @@ _sourceCategory=swssDev
...
+The **Symantec WWeb Security ServiceSS - Web Activity Overview** dashboard provides a comprehensive view of the web activity status through various widgets. It includes widgets for blocked websites by category, blocked user agents, number of blocked sites over time, blocked sites, bytes sent/received over time, Layer 4 result actions, requests by the user, time spent processing requests by the user, bandwidth usage by user, web activity by total hits, web activity by unique users, and top 10 URI extensions. This information helps you to monitor and manage network security in real time, which allows you to quickly identify and respond to potential threats. With Symantec Web Security Service, businesses can ensure a safe browsing experience and protect against various online threats.<br/><img src={useBaseUrl('img/integrations/saas-cloud/Symantec-Web-Security-Service-Web-Activity-Overview.png')} alt="Symantec-Web-Security-Service-Web-Activity-Overview" width="800"/>
```
Comment (jpipkin1) [inline docs/integrations/saas-cloud/symantec-web-security-service.md:83]:
```suggestion
The **Symantec Web Security Service - Web Activity Overview** dashboard provides a comprehensive view of the web activity status through various widgets. It includes widgets for blocked websites by category, blocked user agents, number of blocked sites over time, blocked sites, bytes sent/received over time, Layer 4 result actions, requests by the user, time spent processing requests by the user, bandwidth usage by user, web activity by total hits, web activity by unique users, and top 10 URI extensions. This information helps you to monitor and manage network security in real time, which allows you to quickly identify and respond to potential threats. With Symantec Web Security Service, businesses can ensure a safe browsing experience and protect against various online threats.<br/><img src={useBaseUrl('img/integrations/saas-cloud/Symantec-Web-Security-Service-Web-Activity-Overview.png')} alt="Symantec-Web-Security-Service-Web-Activity-Overview" width="800"/>
```

File: docs/integrations/saas-cloud/workday.md [hunk 7]
```diff
@@ -11,17 +11,15 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
 Workday is a cloud-based ERP system that manages the business processes and allows organizations to use a system integrated application. Workday is a coherent cloud ERP system for financial analysis, analytical solutions, HCM suites, and better business processes.
 
-The Sumo Logic App for Workday provides insights into authentication activity, user activity, and administrator activity. The pre-configured dashboards help identify events that identify compliance and incident reporting, common security events, and real-time analysis of suspicious activities.
+The Sumo Logic app for Workday provides insights into authentication activity, user activity, and administrator activity. The pre-configured dashboards help identify events that identify compliance and incident reporting, common security events, and real-time analysis of suspicious activities.
 
-
-## Log Types
+## Log types
 
 The Sumo Logic App for Workday collects all logs in JSON format. It uses the following two types of logs:
```
Comment (jpipkin1) [inline docs/integrations/saas-cloud/workday.md:18]:
```suggestion
The Sumo Logic app for Workday collects all logs in JSON format. It uses the following two types of logs:
```

File: docs/integrations/saas-cloud/workday.md [hunk 8]
```diff
@@ -11,17 +11,15 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
 * SignOn Logs
 * Activity Logs
 
-
-### Sample Log Messages
+### Sample log messages
 
 Workday App logs are all in JSON format. The Workday App uses two types of logs and this section provides examples of the log types.
```
Comment (jpipkin1) [inline docs/integrations/saas-cloud/workday.md:24]:
```suggestion
Workday app logs are all in JSON format. The Workday App uses two types of logs and this section provides examples of the log types.
```

Comment (jpipkin1) [inline docs/integrations/saas-cloud/workday.md:24]:
```suggestion
Workday app logs are all in JSON format. The Workday app uses two types of logs and this section provides examples of the log types.
```

File: docs/integrations/saas-cloud/workday.md [hunk 9]
```diff
@@ -86,7 +82,7 @@ Sumo Logic collects data from the User Activity and Signon Activity Reports via
 
 User activity data is collected through the Workday Activity Logs API. To ensure that no sensitive information is being sent to Sumo Logic via this report, please run the “User Activity” Report and check the columns (specifically the Target column). If the data contains sensitive info, you can enable data masking for the security group created in the steps outlined below by following the instructions in [this doc](https://doc.workday.com/reader/Z9lz_01hqDMDg6NSf7wCBQ/uHBXsJmAzuJ2QFVU6D3o2w).
 
-## Collecting Logs for the Workday App
+## Collecting Logs for the Workday app
 
 This section explains how to collect logs from Workday and ingest them into Sumo Logic for use with the Workday App predefined dashboards and searches.
```
Comment (jpipkin1) [inline docs/integrations/saas-cloud/workday.md:87]:
```suggestion
This section explains how to collect logs from Workday and ingest them into Sumo Logic for use with the Workday app predefined dashboards and searches.
```

File: docs/integrations/security-threat-detection/f5-big-ip-ltm.md [hunk 10]
```diff
@@ -57,8 +55,7 @@ _sourceCategory=f5/bigip/ltm  systemInfo
 | count by virtualServer_name, pool, application, enabledState, availabilityState
 ```
 
-
-## Collecting Logs for the F5 - BIG-IP LTM App
+## Collecting Logs for the F5 - BIG-IP LTM app
```
Comment (jpipkin1) [inline docs/integrations/security-threat-detection/f5-big-ip-ltm.md:58]:
```suggestion
## Collecting logs for the F5 - BIG-IP LTM app
```

File: docs/integrations/security-threat-detection/f5-big-ip-ltm.md [hunk 11]
```diff
@@ -71,9 +68,7 @@ Telemetry Streaming (TS) is a software RPM file. Installing it on BIG-IP enables
 
 Telemetry Streaming uses a declarative model, meaning you provide a JSON declaration rather than a set of imperative commands. For more information on Telemetry Streaming, see [here](https://clouddocs.f5.com/products/extensions/f5-telemetry-streaming/latest/).
 
-
-
-### Configure Log Collection for the F5 - BIG-IP LTM App
+### Configure Log Collection for the F5 - BIG-IP LTM app
```
Comment (jpipkin1) [inline docs/integrations/security-threat-detection/f5-big-ip-ltm.md:71]:
```suggestion
### Configure log collection for the F5 - BIG-IP LTM app
```

File: docs/integrations/security-threat-detection/imperva-incapsula.md [hunk 12]
```diff
@@ -68,9 +67,7 @@ _sourceCategory="Incapsula"
 | count by policy_type
 | top 10 policy_type by _count
 ```
-
-
-## Collecting Logs for the Imperva-Incapsula WAF App
+## Collecting Logs for the Imperva-Incapsula WAF app
```
Comment (jpipkin1) [inline docs/integrations/security-threat-detection/imperva-incapsula.md:70]:
```suggestion
## Collecting logs for the Imperva-Incapsula WAF app
```

File: docs/integrations/security-threat-detection/netskope.md [hunk 13]
```diff
@@ -102,35 +101,27 @@ to_user, app_session_id  nodrop
 ```
 
 
-## Installing the Netskope App
+## Installing the Netskope app
 
 This section demonstrates how to install the Netskope App, and provides examples and descriptions for each of the dashboards. The Netskope App has the following components:
```
Comment (jpipkin1) [inline docs/integrations/security-threat-detection/netskope.md:106]:
```suggestion
This section demonstrates how to install the Netskope app, and provides examples and descriptions for each of the dashboards. The Netskope app has the following components:
```

File: docs/integrations/security-threat-detection/netskope.md [hunk 14]
```diff
@@ -13,20 +13,19 @@ The Netskope App provides visibility into the security posture of your applicati
 
 Netskope is a Cloud Access Security Broker (CASB) hosted in the cloud. The Netskope product is primarily used for enforcing security policies for cloud-based resources, such as Box and Microsoft Office 365. Customers purchase a CASB to address cloud service risks, enforce security policies, and comply with regulations, even when cloud services are beyond their perimeter and out of their direct control.
 
-
-## Log Types
+## Log types
 
 The Netskope App provides a collector source for pulling all the events and alerts from Netskope in real-time via API calls and ingests them into the Sumo Logic platform through our Hosted collector.
```
Comment (jpipkin1) [inline docs/integrations/security-threat-detection/netskope.md:18]:
```suggestion
The Netskope app provides a collector source for pulling all the events and alerts from Netskope in real-time via API calls and ingests them into the Sumo Logic platform through our Hosted collector.
```

File: docs/integrations/security-threat-detection/observable-networks.md [hunk 15]
```diff
@@ -21,11 +21,11 @@ For more information, visit [Observable Networks (via Cisco)](https://www.cisco.
 Before you begin, your Observable Networks portal must be properly configured. Contact [support@obsrvbl.com](/docs/integrations/security-threat-detection/observable-networks) if you have any questions.
 :::
 
-## Log Types
+## Log types
 
 The Sumo Logic App for Observable Networks assumes Observable Networks formatted logs, which provide one JSON message per request.
 
-## Collect Logs for the Observable Network App
+## Collect Logs for the Observable Network app
```
Comment (jpipkin1) [inline docs/integrations/security-threat-detection/observable-networks.md:28]:
```suggestion
## Collect logs for the Observable Network app
```

File: docs/integrations/security-threat-detection/observable-networks.md [hunk 16]
```diff
@@ -164,17 +161,16 @@ _sourceCategory=observable
 
 ## Install the Observable Networks App
```
Comment (jpipkin1) [inline docs/integrations/security-threat-detection/observable-networks.md:162]:
```suggestion
## Install the Observable Networks app
```

File: docs/integrations/security-threat-detection/twistlock.md [hunk 17]
```diff
@@ -27,8 +26,7 @@ For Kubernetes customers, we recommend installing the Twistlock App.
 
 For non-Kubernetes customers, we recommend installing the Twistlock Classic App.
 
-
-## Collecting Logs for the Twistlock App
+## Collecting Logs for the Twistlock app
```
Comment (jpipkin1) [inline docs/integrations/security-threat-detection/twistlock.md:29]:
```suggestion
## Collecting logs for the Twistlock app
```

File: docs/reuse/apps/app-install-v2.md [hunk 18]
```diff
@@ -1,15 +1,16 @@
...
     1. **Key.** Select either of these options for the data source.
         * Choose **Source Category**, and select a source category from the list for **Default Value**.
         * Choose **Custom**, and enter a custom metadata field. Insert its value in **Default Value**.
-1. Click **Next**. You will be redirected to the **Preview & Done** section.
+1. Click on **Next**. You will be redirected to **Preview & Done** section.
 
-Your app will be installed in the **Installed Apps** folder and dashboard panels will start to fill automatically.
+Your app will be installed in **Installed Apps** folder and Panels of dashboard will start to fill automatically.
 
 Each panel slowly fills with data matching the time range query and received since the panel was created. Results will not immediately be available, updating with full graphs and charts over time.
```
Comment (jpipkin1) [inline docs/reuse/apps/app-install-v2.md:16]: @JV0812, I don't think you should make any of the changes indicated in this file.

**Review by jpipkin1 — CHANGES_REQUESTED** [review]
@JV0812, I just found small things. 

Please look at my final comment and let me know if you agree that you shouldn't make the changes inidcated in the [docs/reuse/apps/app-install-v2.md](https://github.com/SumoLogic/sumologic-documentation/pull/3433/files/c80077568dbd5e5c69ffc15e697fa4efdda43692#diff-1bc4db5787628829a5743002cbc2363258b835dafd98ec5f11e507af8ef21a7d) file.

File: docs/reuse/apps/app-install-v2.md (hunk 18 above)
Comment (JV0812) [reply docs/reuse/apps/app-install-v2.md:16]: Yes John, we should not do any changes to this file. I will revert this.

**Review by kimsauce — APPROVED** [review]

**Review by jpipkin1 — APPROVED** [review]

**Comment by himanshu219** [comment]
@jpipkin1 @JV0812  could you please resolve these conflicts and merge. Conflicts are coming in formatting of spaces I don't know which ones to keep.

**Comment by kimsauce** [comment]
> @jpipkin1 @JV0812 could you please resolve these conflicts and merge. Conflicts are coming in formatting of spaces I don't know which ones to keep.

Yikes, I'll look into this now

**Comment by jpipkin1** [comment]
@kimsauce, @JV0812 told me he would fix this. He's been in contact with @himanshu219 about it.

**Comment by kimsauce** [comment]
> @kimsauce, @JV0812 told me he would fix this. He's been in contact with @himanshu219 about it.

Thank you for the heads up! I've already fixed it in a new PR. The issues were broken branch name (parens corrupted it) and Docusaurus v2 --> v3 syntax.

**Pull request closed without merging**